### DISCUSSION TITLE
Is Github "Assign up to 10 people to this issue" allowing for private repository in free organize now?

### DISCUSSION THREAD

**Discussion by tuanngocptn** [community/community#126318] [Other Feature Feedback, Questions, & Ideas]
Hello everyone.

I have the strange thing after I took over the Github Org from my friend. That allow me assign issue to multiple people in private repository without PRO plan!

I checked for multiple source in google for finding the answer of this thing. But everyones said that need PRO plan for private repository in Free plan ORG.

Anyone can explain me why this ORG can do that. And how to do that in other ORG. Because I really want this feature for other ORG that I'm owner ?

Thanks so much!

*you can see the image below here for more information*

- The "Assign up to 10 people to this issue" is allow in private repository in Free plan ORG:

![image](https://github.com/community/community/assets/22292704/b567f282-191a-409e-a459-ee3db5838450)

- And this is the plan of that Org:

![image](https://github.com/community/community/assets/22292704/19b8068c-359e-412e-9d1c-7093be9bbc39)

**Comment by tebitobi** [comment]
Did you get any response from Github support?
